### ISSUE TITLE
is there a way to support Serializers without route handler?

### ISSUE THREAD

**Issue by Daniel-Xu** [samselikoff/ember-cli-mirage#647]
Right now, I just use `ember-cli-mirage` in Integration test, and I want to use `create` and `creatList` methods.

The problem is: it always returns JSON-API formatted data even I setup the application serializer and model serializer.

My factory looks like this:

```
import { Factory, faker } from 'ember-cli-mirage';

export default Factory.extend({
  depth: 1,
  order(i) { return `${i}` },
  title: () => faker.lorem.sentence(),
  kind: 'StructureNode',
  parent_id() { return 100 * Math.random() % 100 },
  children: []
});
```

**Comment by samselikoff** [comment]
Can you show me an example of what you'd like to accomplish, and what you're trying now that's not working?

**Comment by Daniel-Xu** [comment]
I tried to create  instances of node using:
`let nodes = server.createList('node',  10);`

The instances returned by `createList` are in JSONAPI data format. However, I just want the same flat format as defined in node factory. 

Now I have a workaround like this: 
 `let nodes = server.db.nodes.map(node => StructureNode.create(node));`

**Comment by samselikoff** [comment]
Hm, `createList` should return `Mirage.Model` instances, depending on what version of Mirage you're on.

The purpose of the factories are to seed your Mirage database, how are you using Mirage in the integration test? Does the test make HTTP requests?

**Comment by Daniel-Xu** [comment]
Actually, the integration test is not making any request now. We try to change all of our tests by using ember-cli-mirage. This one is just the starting point. 

From the docs, I saw that the serializer works only inside of route handler. (and I did try to change the application serializer and model serializer). The version I'm using is 0.2.0-beta7 :heart: 

> Any Model or Collection returned from a route handler will pass through the serializer layer. Highest > priority will be given to a model-specific serializer, then the application serializer, then the default > serializer.
